crates/swc_ecma_transforms_base/src/helpers/mod.rs

```diff
@@ -321,17 +323,7 @@ impl VisitMut for Marker {
         i.span = i.span.apply_mark(self.0);
     }
 
-    fn visit_mut_member_prop(&mut self, p: &mut MemberProp) {
-        if let MemberProp::Computed(p) = p {
-            p.visit_mut_with(self);
-        }
-    }
-
-    fn visit_mut_super_prop(&mut self, p: &mut SuperProp) {
-        if let SuperProp::Computed(p) = p {
-            p.visit_mut_with(self);
-        }
-    }
+    visit_mut_obj_and_computed!();
```

Comment:
We don't visit `obj` here.

https://github.com/swc-project/swc/blob/b4f268fa23f90becfea42783e1dd3505fc115d0b/crates/swc_ecma_transforms_base/src/helpers/mod.rs#L324-L328